JSDemos/Demos/Toast/Overview/Angular/app/app.component.html

```diff
@@ -1,13 +1,16 @@
-<div id="toast">
+<div id="productList">
     <h1>Product List</h1>
     <ul>
         <li *ngFor="let product of products">
-            <img [src]="product.ImageSrc">
+            <img [src]="product.ImageSrc" />
             <div id="name">{{product.Name}}</div>
```

Comment:
I just noticed that all divs have the same "id" -> name. Though it doesn't break anything and same ids look bad. This logic comes from the current demo and it's better to correct this everywhere (Angular, React, Vue, jQuery, ASP.NET)

Reply:
it looks like that this ID is not necessary at all. I'm going to remove it.